src/validations/index.js

```diff
@@ -1,4 +1,11 @@
+var i18n = require('i18next-client');
+i18n.init({
```

Comment:
It looks like `i18next` supports a [`resStore` argument](http://i18next.com/pages/doc_init.html) that would allow us to bundle the translations into the main browserify bundle, but in this case it’s probably better to use the current approach. Worth loading another file to avoid downloading a bunch of translations in languages we don’t need